### ISSUE TITLE
parseVtepMac doesn't properly parse flannel.alpha.coreos.com/backend-data: {"VNI":4096,"VtepMAC":"00:15:5d:b3:ac:db"}

### ISSUE THREAD

**Issue by ssthomas** [F5Networks/k8s-bigip-ctlr#1480]
#### Setup Details
 CIS Version : 2.X      
 Build: f5networks/k8s-bigip-ctlr:latest       
 BIGIP Version: Big IP x.x.x      
 AS3 Version: 3.x      
 Agent Mode: AS3/CCCL       
 Orchestration: K8S/OSCP      
 Orchestration Version:       
 Pool Mode: Cluster/Nodeport      
 Additional Setup details: <Platform/CNI Plugins/ cluster nodes/ etc>

#### Description  

parseVtepMac parses JSON data as a string rather than JSON. As a result, valid JSON such as   {"VNI":4096,"VtepMAC":"00:15:5d:b3:ac:db"} returns "VtepMAC" as the MAC address

#### Steps To Reproduce
1) add windows worker node with flannel
2) Use a non-default VNI (e.g. 4096)
3) Attempt to configure vxlan tunnel between f5 and Kubernetes cluster

#### Expected Result 

parseVtepMac returns a mac address

#### Actual Result  

parseVtepMac returns whatever the third field is when split() with "

#### Diagnostic Information
```buildoutcfg
<Configuration files, error messages, logs>
Note: Sanitize the data. For example, be mindful of IPs, ports, application names and URLs
Note: The following F5 article outlines the information required when opening an issue.
https://support.f5.com/csp/article/K60974137
```  

#### Observations (if any)

something like this will probably be close, but I don't really speak Golang. So, I guarantee it will not compile or work.

func parseVtepMac(mac, nodeName string) (string, error) {
        var result map[string]interface{}
        json.Unmarshal([]byte(mac), &result)
        vtep_mac := result["VtepMAC"].(map[string]interface{})
        key, mac_addr := vtep_mac
        if mac_addr == nil {
                err := fmt.Errorf("flannel.alpha.coreos.com/backend-data annotation for "+
                        "node '%s' has invalid format; cannot validate VtepMac. "+
                        "Should be of the form: '{\"VtepMAC\":\"<mac>\"}'", nodeName)
                return "", err
        } else {
                return mac_addr.(string), nil
        }
}

**Comment by cisbotctlr** [comment]
cisbot will assign the issue to one of the devs. 
 @devs, use /jira for internal tracking.

**Comment by ssthomas** [comment]
This is related to SR C3364277

**Comment by mdditt2000** [comment]
@ssthomas Windows support is now available in the latest release of flannel RC 14.0-rc1

quay.io/coreos/flannel:v0.14.0-rc1

Also CIS team fixed the mac addressing parsing issue in CIS 2.4 https://github.com/F5Networks/k8s-bigip-ctlr/releases/tag/v2.4.0

Please can re-test. Thanks you. This issue can be closed.
